Doc/c-api/unicode.rst

```diff
@@ -1502,3 +1502,83 @@ They all return ``NULL`` or ``-1`` if an exception occurs.
    :c:func:`PyUnicode_InternInPlace`, returning either a new Unicode string
    object that has been interned, or a new ("owned") reference to an earlier
    interned string object with the same value.
+
+PyUnicodeWriter
+^^^^^^^^^^^^^^^
+
+The :c:type:`PyUnicodeWriter` API can be used to create a Python :class:`str`
+object.
+
+.. versionadded:: 3.14
+
+.. c:type:: PyUnicodeWriter
+
+   An Unicode writer instance.
```

Comment:
This should be "**A** Unicode" and not "**An**" (this is also consistent with the rest of the docs and this is because "Unicode" starts with a consonant sound even though U is a vowel).